colrev/ops/built_in/search_sources/aisel.py

```diff
@@ -380,17 +382,66 @@ def get_masterdata(
     def load(self, load_operation: colrev.ops.load.Load) -> dict:
         """Load the records from the SearchSource file"""
 
+        enl_mapping = {
+            ENTRYTYPES.ARTICLE: {
+                "T": Fields.TITLE,
+                "A": Fields.AUTHOR,
+                "D": Fields.YEAR,
+                "B": Fields.JOURNAL,
+                "V": Fields.VOLUME,
+                "N": Fields.NUMBER,
+                "P": Fields.PAGES,
+                "X": Fields.ABSTRACT,
+                "U": Fields.URL,
+                "8": "date",
+                "0": "type",
+            },
+            ENTRYTYPES.INPROCEEDINGS: {
+                "T": Fields.TITLE,
+                "A": Fields.AUTHOR,
+                "D": Fields.YEAR,
+                "B": Fields.JOURNAL,
+                "V": Fields.VOLUME,
+                "N": Fields.NUMBER,
+                "P": Fields.PAGES,
+                "X": Fields.ABSTRACT,
+                "U": Fields.URL,
+                "8": "date",
+                "0": "type",
+            },
+        }
+
+        entrytype_map = {
```

Comment:
Not sure this would work because the %0 field does not contain "HICCS", ... . I believe the fix_entrtype_before_conversion should be sufficient.

Reply:
Hm, I didn't realize! Updated